R/extract-transform-sherlock.R

```diff
@@ -180,7 +180,7 @@ process_well_sample_details <- function(filepath, sample_type, layout_type, sing
   sample_type_id <- ifelse(sample_type == "mucus", 1, 2)
 
   layout_raw <- suppressMessages(read_excel(filepath,
```

Comment:
make sure to add `readxl::` to `read_excel` I've been catching places where I also forgot, but moving forward let's make sure we add these to avoid some global env bugs